build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1690,4 +1690,18 @@ message RequestMetadata {
   // An identifier to tie multiple tool invocations together. For example,
   // runs of foo_test, bar_test and baz_test on a post-submit of a given patch.
   string correlated_invocations_id = 4;
+
+  // A one-word description of the kind of action, for example, CppCompile or GoLink.
+  // There is no standard agreed set of values for this, and they are expected to vary between different client tools.
+  string action_mnemonic = 5;
+
+  // An identifier for the target which produced this action.
+  // No guarantees are made around how many actions may relate to a single target.
+  string target = 6;
```

Comment:
The other similar fields all use the suffix 'id' so I think this should be target_id.

Reply:
Done